src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -256,11 +255,19 @@ function useChatUtilsHook(): UseChatUtilsHook {
   };
 
   const goBackFromChatScreen = async () => {
-    navigation.goBack();
-    setChat((prevChat) => ({
-      ...prevChat,
-      selectedChannel: null
-    }));
+    if (type === 'ANONYMOUS') {
+      navigation.navigate('AnonymousChannelList');
+    }
+    if (type === 'SIGNED') {
+      navigation.navigate('SignedChannelList');
+    }
```

Comment:
ini akan ngerubah history screennya ga ya bang @asaadam ? contohnya misal scroll channel list nya sampe agak bawah, terus buka satu channel, terus dia trigger function `goBackFromChatScreen` channel listnya balik ke top of list ga? kalau `goBack()` seingetku engga ngerubah history nya

Reply:
dia tetep ada dibawah, udh tak coba sih. tak kirim di slack ya

Reply:
oke, harusnya aman sii

Reply:
eh sek ekspektasinya ke atas, apa sesuai terakhir?

Reply:
sesuai posisi terakhir harusnyaa